my-react-app/src/app/screens/Login/components/Fields/customInput.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
```

Comment:
Move this file to `src/components/CustomInput/index.js` (keep in mind the folder where you place the component should start in Uppercase and the component itself should be called `index.js`)